src/components/Main/main.scss

```diff
@@ -5,9 +5,15 @@
     width: 100%;
 }
 
-#login-page {
+#landing-layout {
+    padding: 0 initial;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
     header {
         padding: size(5) 0;
+        text-align: -webkit-center;
```

Comment:
why are you using this -webkit-center? @booleanrecep

Reply:
That was intended to align the content to the center but I see it's useless. So I deleted it. Thanks for your attention.